### PR TITLE
Add `restart-all-ecs-tasks` command

### PR DESCRIPTION
**Pull request by R4c00n**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# restart-all-ecs-tasks
+
+> CLI tool to restart all ECS tasks inside a cluster
+
+This project was born out of the need to restart all ECS tasks automatically, without having duplicate service definitions.
+ 
+The commend restarts all tasks inside a given cluster, one at a time. After stopping one tasks it waits until the ECS service
+has started a new tasks before it stops the next one.
```
Comment (Pupskuchen) [inline README.md]: unnecessary line break

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+You can restart all ECS tasks on a given cluster with the `restart-all-ecs-tasks` command:
+
+```
+$ restart-all-ecs-tasks -c arn:aws:ecs:region:1234567890123:cluster/environment
+```
+
+## Binary path
+
+By default the `restart-all-ecs-tasks` commands executes the `aws` binary. If your environment is not configured properly 
+and you need to specify the full path to the `aws` binary you can specify it using the `-b` or `--binary` option.
```
Comment (Pupskuchen) [inline README.md]: another unnecessary line break

File: Ecs.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        }
+    }
+
+    /**
+     * Get a list of all tasks on the cluster
+     *
+     * @returns {string[]} - Array of task ARNs
+     * @private
+     */
+    _listTasks() {
```
Comment (Pupskuchen) [inline Ecs.js:50]: Correct me if I'm wrong but I think the mediaman JS styleguide says not to use trailing or leading underscores (even for "private" properties)

**Review by Pupskuchen — CHANGES_REQUESTED** [review]

File: Ecs.js (hunk 3 above)
Comment (R4c00n) [reply Ecs.js:50]: The mm JS style guide is currently only applicable for ES5, so I think we need to discuss this on a different occasion.

File: README.md (hunk 1 above)
Comment (R4c00n) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (R4c00n) [reply README.md]: Done

**Review by Pupskuchen — APPROVED** [review]

**Pull request merged**